autospec/git.py

```diff
@@ -54,6 +54,7 @@ def commit_to_git(path):
     call("git add make_prepend", check=False, stderr=subprocess.DEVNULL, cwd=path)
     call("git add install_prepend", check=False, stderr=subprocess.DEVNULL, cwd=path)
     call("git add install_append", check=False, stderr=subprocess.DEVNULL, cwd=path)
+    call("git add service_restart", check=False, stderr=subprocess.DEVNULL, cwd=path)
```

Comment:
This isn't required because read_conf_file adds it to the config_files list and that entire list is added above.

Reply:
Ah, thanks for the catch. I have fixed this.